assets/js/base/components/cart-checkout/order-summary/style.scss

```diff
@@ -27,6 +27,8 @@
 		}
 
 		&:last-child {
+			border-width: 0;
```

Comment:
Is this to reset the style above? I would prefer to see `border-bottom: 0` if so, just resetting what you added before.